### PR TITLE
Finalinventory

### PR DESCRIPTION
**Pull request by chiually**
inventory use case complete

### CODE REVIEW

File: src/main/java/app/builder/DataAccessBuilder.java [hunk 1]
```diff
@@ -63,7 +70,9 @@ private DataAccessBuilder buildUserDataAccess() {
      * @return this builder
      */
     private DataAccessBuilder buildInventoryDataAccess() {
-        this.inventoryDataAccess = factory.createInventoryDataAccess();
+        final DBInventoryDataAccessObject dbInventoryDataAccess =
+                new DBInventoryDataAccessObject(new FoodInventoryFactory(), new FoodItemFactory());
```
Comment (rachelkd) [inline src/main/java/app/builder/DataAccessBuilder.java]: Make a createInventoryDataAccess method in ViewFactory instead. Shouldn't be creating new objects in this file

Comment (rachelkd) [reply src/main/java/app/builder/DataAccessBuilder.java]: Then, you can remove the entity imports in this class

File: src/main/java/app/InventoryExample.java [hunk 2]
```diff
@@ -58,40 +57,34 @@ public static void main(String[] args) {
     }
 
     public JFrame build() {
```
Comment (rachelkd) [inline src/main/java/app/InventoryExample.java:60]: Missing Javadoc

File: src/main/java/app/InventoryExample.java [hunk 3]
```diff
@@ -58,40 +57,34 @@ public static void main(String[] args) {
     }
```
Comment (rachelkd) [inline src/main/java/app/InventoryExample.java:58]: Class Data Abstraction Coupling is 19 (max allowed is 15). Make helper methods in this file

File: src/main/java/view/InventoryView.java [hunk 4]
```diff
@@ -107,32 +158,32 @@ public void setChangeCatHungerController(ChangeCatHungerController changeCatHung
...
-                        if (selectedLabel[0] != null) {
-                            final String selectedText = selectedLabel[0].getText();
-                            JOptionPane.showMessageDialog(null, "Selected Item: " + selectedText);
-                        }
-                        else {
-                            JOptionPane.showMessageDialog(null, "Please select an item first.");
-                        }
-                    });
+            buttonPanel.add(selectItemButton, BorderLayout.SOUTH);
+            selectItemButton.addActionListener(event -> {
```
Comment (rachelkd) [inline src/main/java/view/InventoryView.java:170]: Lambda body length is 11 lines (max allowed is 10). Make helper method and call it in your lambda function

**Review by rachelkd — CHANGES_REQUESTED** [review]
Just refactoring to be done.
Mostly just Checkstyle fixes that can be resolved by making helper methods. Also, DataAccessBuilder should be using the DataAccessFactory instead of creating a new DAO in the file.

File: src/main/java/app/InventoryExample.java (hunk 2 above)
Comment (chiually) [reply src/main/java/app/InventoryExample.java:60]: this file will be deleted in the end ignore checkstyle

File: src/main/java/app/InventoryExample.java (hunk 3 above)
Comment (chiually) [reply src/main/java/app/InventoryExample.java:58]: file will be deleted

File: src/main/java/app/builder/DataAccessBuilder.java (hunk 1 above)
Comment (chiually) [reply src/main/java/app/builder/DataAccessBuilder.java]: In latest commit

**Review by rachelkd — APPROVED** [review]
looks good

**Pull request merged**